.github/workflows/testworkflow.yml

```diff
@@ -0,0 +1,60 @@
+name: testworkflow
+
+on: # run any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'releases/*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  testworkflow:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: echo 'https://github.com/fair-software/howfairis' > urls.txt
+      - uses: tortellini-tools/action@main
+        with:
+          repositories: urls.txt
+
+      - run: ls -l .tortellini/out/fair-software/howfairis/gl-license-scanning-report.json
+     
+      - run: |
+          git clone --depth 1 https://github.com/oss-review-toolkit/ort.git /tmp/ort
+          DOCKER_BUILDKIT=1 docker build -t ort /tmp/ort
+          mkdir -p conf
+          curl -L https://github.com/oss-review-toolkit/ort/raw/master/examples/rules.kts > conf/rules.kts
+          curl -L https://github.com/oss-review-toolkit/ort/raw/master/examples/license-classifications.yml > conf/license-classifications.yml
+          curl -L https://github.com/oss-review-toolkit/ort/raw/master/examples/curations.yml > conf/curations.yml
+          git clone https://github.com/fair-software/howfairis
+          mkdir .ort
+          docker run --rm -v $PWD/howfairis:/project -v $PWD/.ort:/out ort analyze -i /project -o /out
```

Comment:
not sure what .ort does but I think you meant to mount $PWD/.ort instead of all of $PWD

Reply:
never mind, read it wrong